diesel_cli/tests/print_schema/print_schema_custom_types/print_schema_simple_without_docs/diesel.toml

```diff
@@ -0,0 +1,2 @@
+[print_schema]
```

Comment:
Those files seems to be in the wrong place?
`diesel_cli/tests/print_schema/print_schema_custom_types/print_schema_simple_without_docs/diesel.toml`
seems to be wrong. 
Maybe they belong to 
`diesel_cli/tests/print_schema/print_schema_simple_without_docs/diesel.toml`?